Yoast/wordpress-seo
# Removed legacy license page and added upgrade routine for option

## Summary

This PR can be summarized in the following changelog entry:

* Removes legacy license page and updates the option to the new license page.

## Test instructions

This PR can be tested by following these steps:

* Set `wpseo_license_server_version` to 1 in your database (`wp_options` table).
* Checkout the PR
* Check that the option was updated and that `licenses-legacy.php` view is gone.

Fixes https://github.com/Yoast/wordpress-seo-premium/issues/1328
